[2022-02-02 18:27:20,894] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-02 18:27:20,904] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-02 18:27:20,905] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:27:20,906] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 18:27:20,907] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:27:20,918] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-09-01 00:00:00+00:00
[2022-02-02 18:27:20,924] {standard_task_runner.py:52} INFO - Started process 9350 to run task
[2022-02-02 18:27:20,929] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpa2774qkk', '--error-file', '/tmp/tmpyteh_1se']
[2022-02-02 18:27:20,932] {standard_task_runner.py:77} INFO - Job 328: Subtask download_dataset_task
[2022-02-02 18:27:21,009] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-02 18:27:21,056] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 18:27:21,077] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-09-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-09-01T00:00:00+00:00
[2022-02-02 18:27:21,079] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 18:27:21,082] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-09.csv > /opt/***/yellow_tripdata_2019-09.csv']
[2022-02-02 18:27:21,098] {subprocess.py:85} INFO - Output:
[2022-02-02 18:29:51,009] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 18:29:51,155] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190901T000000, start_date=20220202T182720, end_date=20220202T182951
[2022-02-02 18:29:51,189] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 18:29:51,231] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 13:04:22,634] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-10 13:04:22,650] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-10 13:04:22,651] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 13:04:22,652] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 13:04:22,653] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 13:04:22,671] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-09-01 00:00:00+00:00
[2022-02-10 13:04:22,680] {standard_task_runner.py:52} INFO - Started process 1557 to run task
[2022-02-10 13:04:22,686] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '684', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpbf_28vu2', '--error-file', '/tmp/tmpu52emweh']
[2022-02-10 13:04:22,688] {standard_task_runner.py:77} INFO - Job 684: Subtask download_dataset_task
[2022-02-10 13:04:22,795] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [running]> on host e30f201fae61
[2022-02-10 13:04:22,887] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 13:04:22,915] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-09-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-09-01T00:00:00+00:00
[2022-02-10 13:04:22,919] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 13:04:22,921] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-09.csv > /opt/***/yellow_tripdata_2019-09.csv']
[2022-02-10 13:04:22,939] {subprocess.py:85} INFO - Output:
[2022-02-10 13:06:11,179] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 13:06:11,212] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190901T000000, start_date=20220210T130422, end_date=20220210T130611
[2022-02-10 13:06:11,249] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 13:06:11,303] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 15:36:49,420] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-10 15:36:49,436] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-10 15:36:49,436] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:36:49,437] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 15:36:49,438] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:36:49,454] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-09-01 00:00:00+00:00
[2022-02-10 15:36:49,465] {standard_task_runner.py:52} INFO - Started process 541 to run task
[2022-02-10 15:36:49,468] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '805', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp1ey50eks', '--error-file', '/tmp/tmpwu6rqf9t']
[2022-02-10 15:36:49,470] {standard_task_runner.py:77} INFO - Job 805: Subtask download_dataset_task
[2022-02-10 15:36:49,537] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:36:49,579] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:36:49,606] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-09-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-09-01T00:00:00+00:00
[2022-02-10 15:36:49,607] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:36:49,609] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-09.csv > /opt/***/yellow_tripdata_2019-09.csv']
[2022-02-10 15:36:49,632] {subprocess.py:85} INFO - Output:
[2022-02-10 15:37:17,385] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:37:17,539] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190901T000000, start_date=20220210T153649, end_date=20220210T153717
[2022-02-10 15:37:17,653] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:37:17,732] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:35:27,616] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-10 16:35:27,647] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2022-02-10 16:35:27,649] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:35:27,652] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 16:35:27,655] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:35:27,679] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-09-01 00:00:00+00:00
[2022-02-10 16:35:27,693] {standard_task_runner.py:52} INFO - Started process 444 to run task
[2022-02-10 16:35:27,699] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '942', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpx50fg9s9', '--error-file', '/tmp/tmp02p3pru2']
[2022-02-10 16:35:27,702] {standard_task_runner.py:77} INFO - Job 942: Subtask download_dataset_task
[2022-02-10 16:35:27,827] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-09-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:35:27,952] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:35:28,078] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-09-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-09-01T00:00:00+00:00
[2022-02-10 16:35:28,081] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:35:28,083] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-09.csv > /opt/***/yellow_tripdata_2019-09.csv']
[2022-02-10 16:35:28,105] {subprocess.py:85} INFO - Output:
[2022-02-10 16:37:21,860] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:37:21,974] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20190901T000000, start_date=20220210T163527, end_date=20220210T163721
[2022-02-10 16:37:22,053] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:37:22,144] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
